File: SDKs/Aspose.Tasks-Cloud-SDK-for-Java/src/main/java/com/aspose/tasks/model/ExtendedAttributeItem.java
package com.aspose.tasks.model;

import com.aspose.tasks.model.Link;
public class ExtendedAttributeItem {
  private Link Link = null;
  private Integer Index = null;
  private String FieldName = null;
  /**
	 * getLink
	 * Gets Link
	 * @return Link
	 */
  public Link getLink() {
    return Link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param Link Link
	 */
  public void setLink(Link Link) {
    this.Link = Link;
  }

  /**
	 * getIndex
	 * Gets Integer
	 * @return Index
	 */
  public Integer getIndex() {
    return Index;
  }

	/**
	 * setIndex
	 * Sets Integer
	 * @param Index Integer
	 */
  public void setIndex(Integer Index) {
    this.Index = Index;
  }

  /**
	 * getFieldName
	 * Gets String
	 * @return FieldName
	 */
  public String getFieldName() {
    return FieldName;
  }

	/**
	 * setFieldName
	 * Sets String
	 * @param FieldName String
	 */
  public void setFieldName(String FieldName) {
    this.FieldName = FieldName;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ExtendedAttributeItem {\n");
    sb.append("  Link: ").append(Link).append("\n");
    sb.append("  Index: ").append(Index).append("\n");
    sb.append("  FieldName: ").append(FieldName).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks-Cloud-SDK-for-.NET/src/Com/Aspose/Tasks/Model/ExtendedAttributeItem.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class ExtendedAttributeItem {
    public Link Link { get; set; }

    public int? Index { get; set; }

    public string FieldName { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ExtendedAttributeItem {\n");
      sb.Append("  Link: ").Append(Link).Append("\n");
      sb.Append("  Index: ").Append(Index).Append("\n");
      sb.Append("  FieldName: ").Append(FieldName).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
